wdio.conf.js

```diff
@@ -0,0 +1,209 @@
+exports.config = {
+
+  //
+  // ==================
+  // Specify Test Files
+  // ==================
+  // Define which test specs should run. The pattern is relative to the directory
+  // from which `wdio` was called. Notice that, if you are calling `wdio` from an
+  // NPM script (see https://docs.npmjs.com/cli/run-script) then the current working
+  // directory is where your package.json resides, so `wdio` will be called from there.
+  //
+  specs: [
+    'test/shell-login.js',
+    'test/study-subject-types.js',
+    'test/study-add.js',
+    'test/study-visit.js',
+    'test/study-details.js',
+    'test/logon.js',
```

Comment:
I'm not sure if it's worth having tests like this one and `navigateToAsmt`. If we do keep them they should be moved to the top of the list since they test functionality present in other tests.

Reply:
I will remove logon.js and navigateToAsmt.